<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>751</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tambah Member</string>
     </attribute>
     <widget class="QLabel" name="L4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Level</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_Alamat">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>571</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="L3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Alamat</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_Member_ID">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="L2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nama</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_Username">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="s_Level">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="L1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Member ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_Password">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="L5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="L6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_Nama">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="b_Tambah">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>360</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tambah</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Edit Member</string>
     </attribute>
     <widget class="QLabel" name="L5_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="L1_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Member_ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="L4_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Level</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_Member_ID_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="s_Level_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_Alamat_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>571</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="L6_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="L3_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Alamat</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_Username_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_Nama_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="L2_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nama</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_Password_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="b_Update">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>360</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_Cari">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cari</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Delete Member</string>
     </attribute>
     <widget class="QLineEdit" name="Member_ID">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="L1_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Member_ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_Hapus">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>200</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="b_Back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Member Menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
